### PR TITLE
Update recipes #3

### PR DESCRIPTION
**Pull request by BusbyActual**
Progress #1662

- [x] [Testing React components](https://github.com/avajs/ava/blob/master/docs/recipes/react.md)

Updated links and babel references to match babel 7. I didn't update any grammar so I'm happy to update the French and Spanish docs if you'd like just let me know.

### CODE REVIEW

File: docs/recipes/precompiling-with-webpack.md [hunk 1]
```diff
@@ -33,9 +30,11 @@ module.exports = {
 		rules: [
 			{
 				test: /\.(js|jsx)$/,
+        exclude: /(node_modules|bower_components)/,
```
Comment (sindresorhus) [inline docs/recipes/precompiling-with-webpack.md]: Incorrect indentation

File: docs/recipes/jspm-systemjs.md [hunk 2]
```diff
@@ -27,25 +28,29 @@ You can find more information about setting up Babel with AVA in the [babelrc re
...
 
 You will need to install the [AVA JSPM loader](https://github.com/skorlir/ava-jspm-loader) as a dev dependency.
+Per Babel 7 you will also need to install [@babel/register](https://www.npmjs.com/package/@babel/register).
 
 ```
 $ npm install --save-dev ava-jspm-loader
 ```
 
+```
+npm install @babel/register --save-dev
```
Comment (vadimdemedes) [inline docs/recipes/jspm-systemjs.md]: Commands should start with `$` and flags should come after `install`.

Comment (vadimdemedes) [reply docs/recipes/jspm-systemjs.md]: Could you also update other PRs to address this?

Comment (BusbyActual) [reply docs/recipes/jspm-systemjs.md]: can do

**Comment by BusbyActual** [comment]
Updated prs and removed duplicate commits from prs

File: docs/recipes/react.md [hunk 3]
```diff
@@ -1,6 +1,3 @@
-> **Please note, this recipe has not yet been updated for Babel 7 support in AVA 1.0.**
-
----
```
Comment (novemberborn) [inline docs/recipes/react.md]: This line ought to be removed as well.

File: docs/recipes/react.md [hunk 4]
```diff
@@ -10,19 +7,23 @@ Translations: [Español](https://github.com/avajs/ava-docs/blob/master/es_ES/doc
 
 The first step is setting up Babel to transpile JSX code from the tests by adding an AVA section to your package.json:
 
+```console
+$ npm install --save-dev @babel/preset-react
+```
```
Comment (novemberborn) [inline docs/recipes/react.md]: It's odd to have this section here. There's no instructions on `@babel/register` either. Perhaps just leave it out, so it's like it was before?

File: docs/recipes/react.md [hunk 5]
```diff
@@ -10,19 +7,23 @@ Translations: [Español](https://github.com/avajs/ava-docs/blob/master/es_ES/doc
...
 	},
 	"babel": {
-		"presets": ["react"]
+		"presets": ["@babel/preset-react"]
 	}
 }
 ```
 
-You can find more information about setting up Babel with AVA in the [babelrc recipe](babelrc.md).
+You can find more information about setting up Babel with AVA in the [babelrc recipe](babel.md).
```
Comment (novemberborn) [inline docs/recipes/react.md]: Could you change this to "Babel recipe", rather than "babelrc recipe"?

**Comment by BusbyActual** [comment]
Merged changes ; ^)

**Comment by novemberborn** [comment]
@BusbyActual I've pushed some tweaks. Notably AVA now by default extends the project configuration so normally no further setup is necessary. What do you think?

**Pull request merged**